src/search_algo.hpp

```diff
@@ -1215,10 +1125,10 @@ iterateMatchesFullSerial(TLocalHolder & lH)
 
     auto const trueQryId = lH.matches[0].qryId / qNumFrames(TGlobalHolder::blastProgram);
 
-    TBlastRecord record(lH.gH.qryIds[trueQryId]);
+    TBlastRecord record(lH.qryIds[trueQryId]);
     record.qLength = seqan::length(lH.gH.qrySeqs[trueQryId]);
```

Comment:
This needs to be ```record.qLength = seqan::length(lH.qrySeqs[trueQryId]);```

Reply:
Yes, that's artifact from merging the branches.